packages/ui-react/src/components/EditPasswordForm/EditPasswordForm.tsx

```diff
@@ -36,13 +37,18 @@ const EditPasswordForm: React.FC<Props> = ({
   value,
   errors,
   submitting,
+  validationError,
   email,
   onResendEmailClick,
 }: Props) => {
   const { t } = useTranslation(['account', 'user']);
   return (
     <form onSubmit={onSubmit} data-testid={testId('forgot-password-form')} noValidate className={styles.forgotPasswordForm}>
-      {errors.form && <FormFeedback variant="error">{errors.form}</FormFeedback>}
+      {errors.form && (
+        <FormFeedback variant="error" visible={!validationError}>
```

Comment:
Hm, I wonder if we it makes sense have a <Form /> component which could include FormFeedback. That way we won't need to add it individually every time.